tests/integration/views/test_static.py

```diff
@@ -17,7 +17,7 @@ def test_contact(self, mock_contact):
 
     def test_contact_with_no_metadata(self):
```

Comment:
This test is no longer valid and the new tests should cover whether session_data is available or not.